### PR TITLE
[Debug] Tell that the extension is for PHP 5 only

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | - |
| License | MIT |
| Doc PR | - |

### CODE REVIEW

File: src/Symfony/Component/Debug/Resources/ext/README.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
-Symfony Debug Extension
-=======================
+Symfony Debug Extension for PHP 5
+=================================
 
-This extension publishes several functions to help building powerful debugging tools.
+This extension publishes several functions to help building powerful debugging tools on PHP
```
Comment (xabbuh) [inline src/Symfony/Component/Debug/Resources/ext/README.md]: missing dot at the end of the line

**Comment by Tobion** [comment]
Forr my information, what PHP 7 functions made this extension obsolete?

**Comment by nicolas-grekas** [comment]
I was thinking about Error that replace fatal errors and have a backtrace now, and about ref mismatches, which are almost gone in php7 and were prevented by this ext also. The other feat are minor ones (not required for VarDumper which is the only referenced use case I know about)

**Comment by fabpot** [comment]
Thank you @nicolas-grekas.

**Pull request merged**